app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -116,6 +118,7 @@
     private int pokemonHP;
     private boolean pokeFlyRunning = false;
     private int trainerLevel;
+    private LruCache<String, String> OCRCache = new LruCache<>(200);
```

Comment:
This is not a final patch, just to get opinion. It was initially called "pokenameOCRCache" and then I decided to combine it into one image cache instead of one per image type.

I used to use "Integer" before, but switched to string because I had to add the "name", "candy", "hp" prefix to the key to avoid the theoretical possibility of a hash collision between a name image and a hp image.